app/javascript/css/tailwind.css

```diff
@@ -0,0 +1,63 @@
+/**
+ * This injects Tailwind's base styles, which is a combination of
+ * Normalize.css and some additional base styles.
+ *
+ * You can see the styles here:
+ * https://github.com/tailwindcss/tailwindcss/blob/master/css/preflight.css
+ *
```

Comment:
so many comments

Reply:
yes and they are going to stay there for now